app/views/users/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here is a specific user</h1>
\ No newline at end of file
+  <%= render partial: "shared/person_info" %>
+  
+  <section class="bio_section">
+    <h1 class="bio_header">Bio</h1>
+    <p class="bio_paragraph">
+      <%= @user.bio %>
+    </p>
+  </section>
+  
+  <section class="posts_section">
+    <% @user.posts.order(created_at: :asc).limit(3).each_with_index do |post, i| %>
+      <div class="post">
+        <span class="title_content">
+          <h2 class="post_title">Post #<%=i + 1%></h2>
```

Comment:
- [ ] A part of the post, preferably the title, on the user show page should be a link that takes the user to the post. Kindly add this link.